### PR TITLE
Fix(frontend): display LTR mention in RTL interface

### PR DESCRIPTION
**Pull request by Haskaris**
Issue ref:
#5324

This PR also fix:
 #3639
 
This PR also bring:
 - Mix LTR and RTL in textarea (comment section)
 - Mix LTR and RTL in comment/actualities (see HTMLContent.styl / html-doc p css)
 - Remove left and right paragraph in tinyMCE
 - Alignment LTR (when displayed Tracim in RTL) for email and username into admin userprofile
 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_app_admin_workspace_user/src/css/index.styl [hunk 1]
```diff
@@ -145,6 +145,11 @@
       width 15%
     &__administrator
       width 15%
+    &__line
+      &__username
+        unicode-bidi plaintext
+      &__email
+        unicode-bidi plaintext
```
Comment (gbonaspetti) [inline frontend_app_admin_workspace_user/src/css/index.styl:152]: I think the e-mails should be left-align even if they start with arabic characters

**Review by gbonaspetti — APPROVED** [review]

**Comment by gbonaspetti** [comment]
The only thing that I found during tests was:
- When I'm on a simple edition if I make a mention or an internal link, I have to confirm two times and I do not have the autocomplete if I want to make a second mention after
mentions

All the rest works really well.

**Comment by gbonaspetti** [comment]
After fix the autocomplete works but we don't have an space added if we are on advanced edition. I think that is acceptable for this version.

**Comment by raphj** [comment]
> This PR also fix:
> https://github.com/tracim/tracim/issues/3639

Fantastic! :-)

> Remove left and right paragraph in tinyMCE

Why?

**Comment by Haskaris** [comment]
The PR brings the detection of LTR and RTL in `textarea` and in comment
Here is why it makes the buttons left/right paragraph not working: writing in Latin characters in right paragraph will stick the text on the left in a comment.

The reason is that left/right paragraph of TinyMCE is creating a `<div dir=ltr/rtl>` into the code. However the detection is done with `unicode-bidi: plaintext` which override the direction.

To make it works (in a cleaner way) we have to encode the direction in `textarea` so we can safely remove `unicode-bidi: plaintext`of every comments.

**Pull request merged**